crates/core/src/db/relational_db.rs

```diff
@@ -370,11 +369,7 @@ impl RelationalDB {
 
     /// Roll back transaction `tx` if `res` is `Err`, otherwise return it
     /// alongside the `Ok` value.
-    #[tracing::instrument(skip_all)]
```

Comment:
I'd keep the `instrument` bit.

Reply:
This is the only thing to fix before landing, then, I think.

Reply:
IMO it's really unnecessary - it's a utility function that we have no need to track and only serves to clutter up logs/graphs. If you feel strongly about it I can add it back, but I really see no value in it.

Reply:
It's not that I do or don't see the value in it. It's that it is unrelated to the stated point of the PR.